protonmail/messages.go

```diff
@@ -403,7 +403,7 @@ type MessagePackageSet struct {
 	Body      string // Encrypted body data packet
 
 	// Only if cleartext is sent
-	BodyKey        string
+	BodyKey        map[string]string
```

Comment:
This should be a struct:

```go
BodyKey *MessageBodyKey `json:omitempty`
```

With a new type `MessageBodyKey` containing the appropriate fields (`Algorithm`, `Key`).